Contributing.md

```diff
@@ -0,0 +1,57 @@
+# Branch Structure
+* **master**: The latest official GA version, which is currently 5.x
+* **dev**: The dev working branch of master, which is 5.x
+* **dev4x**: The dev working branch of master for 4.x
```

Comment:
Should this text say 
**dev4x**: The dev working branch for 4.x
So we don't confuse people into thinking it's related to the master branch?